Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/HandlesBase.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
     /// </summary>
     public abstract class HandlesBase : IProximityEffectObjectProvider
     {
+        static protected string visualsName = "visuals";
```

Comment:
Nit: I think you'll want this to be const, unless we intend to have this be changed by some subclass